modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/construction/ConstructionService.scala

```diff
@@ -3,25 +3,40 @@ package org.tessellation.rosetta.domain.construction
 import cats.data.EitherT
 import cats.effect.Async
 import cats.syntax.applicativeError._
+import cats.syntax.either._
+import cats.syntax.flatMap.toFlatMapOps
```

Comment:
We tend to import whole modules so `cats.syntax.flatMap._`

Reply:
Changed.